src/client/inputhandler.h

```diff
@@ -192,6 +211,14 @@ class RealInputHandler : public InputHandler
 	{
 		return m_receiver->WasKeyDown(keyCode);
 	}
+	virtual void listenForKey(const KeyPress &keyCode)
```

Comment:
those functions doesn't need to be virtual. There is not inheritance from RealInputHandler, no ?